test/WebJobs.Script.Tests.E2E/FunctionAppFixture.cs

```diff
@@ -0,0 +1,193 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.IO;
+using System.IO.Compression;
+using System.Linq;
+using System.Net.Http;
+using System.Threading.Tasks;
+using Microsoft.ApplicationInsights;
+using Microsoft.ApplicationInsights.DataContracts;
+using Xunit;
+
+namespace WebJobs.Script.EndToEndTests
+{
+    public class FunctionAppFixture : IDisposable
```

Comment:
Can you create a static Guid here that would let you log the same "TestRunId" accross all of the other traces? I haven't seen the data yet (so I may be missing it), but I didn't see a way to correlate everything together.